src/style.css

```diff
@@ -1,3 +1,133 @@
-body {
+* {
   margin: 0;
 }
+
+body {
+  font-family: "Roboto", sans-serif;
+}
+
+.card {
+  box-sizing: border-box;
+  width: 200px;
+  padding: 32px 16px 16px;
+
+  background: #fff;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+}
+
+.image {
+  margin-left: 3px;
+}
+
+.title {
+  margin-top: 36px;
+
+  font-size: 12px;
+  line-height: 18px;
+  font-weight: 500;
+
+  color: #060b35;
+}
+
+.prodcode {
+  margin-top: 4px;
+
+  font-weight: 400;
+  font-size: 10px;
+  line-height: 14px;
+
+  color: #616070;
+}
+
+.review {
+  margin-top: 16px;
+
+  display: flex;
+  justify-content: space-between;
+
+  font-weight: 400;
+  font-size: 10px;
+  line-height: 14px;
+  align-items: flex-end;
+
+  color: #060b35;
+}
+
+.stars {
+  display: flex;
+}
+
+.stars__star {
+  width: 16px;
+  height: 16px;
+  background: url(./images/star.svg);
+  background-repeat: no-repeat;
+  background-position: center;
+}
+
+.stars__star:not(:last-child) {
+  margin-right: 4px;
+}
+
+.stars--0 .stars__star:nth-child(-n),
+.stars--1 .stars__star:nth-child(-n + 1),
+.stars--2 .stars__star:nth-child(-n + 2),
+.stars--3 .stars__star:nth-child(-n + 3),
+.stars--4 .stars__star:nth-child(-n + 4),
+.stars--5 .stars__star:nth-child(-n + 5) {
+  background: url(./images/star-active.svg) no-repeat center;
+}
+
+.price {
+  margin-top: 24px;
+
+  display: flex;
+  justify-content: space-between;
+}
+
+.price-title {
+  font-weight: 400;
+  font-size: 12px;
+  line-height: 18px;
+
+  color: #616070;
+}
+
+.price-value {
+  font-weight: 700;
+  font-size: 16px;
+  line-height: 18px;
+
+  color: #060b35;
+}
+
+.buybutton {
+  margin-top: 16px;
+}
+
+.buylink {
+  display: block;
+  padding: 12px 69.5px;
+
+  text-decoration: none;
+  background: #00acdc;
+  border-radius: 5px;
+  transition: all 0.3s ease;
+
+  font-weight: 700;
+  font-size: 14px;
+  line-height: 16px;
+  text-align: center;
+  text-transform: uppercase;
+
+  color: #fff;
+}
+
+.buylink:hover {
+  background: #fff;
+  border: 1px solid #00acdc;
+  padding: 11px 68.5px;
```

Comment:
you can use the box-sizing: border-box; rule to make the border width included in the overall width of the element
 https://developer.mozilla.org/en-US/docs/Web/CSS/box-sizing